src/lv2vst.h

```diff
@@ -72,7 +72,7 @@ class Lv2VstUI : public VstGui
 		virtual ~Lv2VstUI ();
 
 		virtual bool get_rect (ERect** rect);
-		virtual bool open (void* ptr);
+		virtual bool open (void* ptr, float scale_factor);
```

Comment:
Is there a reason why you use an argument for this, instead of a private variable like sample-rate, etc?

Reply:
because that value is not known when the UI object is created (since ui is created together with the DSP/plugin side).
Also this is only needed during `open`.
We could have a `set_scale_factor` function in the UI side I guess. I was not sure how to best go about this. feel free to change of course.